### DISCUSSION TITLE
Contributing by testing on new model

### DISCUSSION THREAD

**Discussion by ggirodda** [Hypfer/Valetudo#1845] [Q&A / Support] [answered]
Hi,
i know that I need to buy a supported model to successfully install and use Valetudo, I read the documentation. I also read that Valetudo is not a custom firmware. But... 
We recently bought a Rowenta explorer 45 series, and I think that it's one of my worst purchases in a long time. I didn't know about Valetudo before searching on the web on how I could replace the vacuum firmware with something better.
Do you think that I can test Valetudo on my vacuum ? i accept any risk, even if the vacuum will stop working. I don't know yet if the model I bought is rootable, neither if Valetudo could be compatible with my model. If there's no chance, because someone already knows the model and already tried it, I will forget it and buy another model at some time.
My Vacuum neither have the map options, that's why I wanted to try Valetudo. But it's possible that the necessary components for this option to work are missing.
I'm a web developer, and I already worked also on IoT devices (long time ago, with Arduino and Raspberry), so i could contribute on the project and add this model to the list of compatible models if this is faisable.
Thank you for any feedback!

**Comment by Hypfer** [comment, marked as answer]
I understand that the situation you're in is frustrating, however Valetudo unfortunately can't be a solution to that even though it would've been great if it was.

As you've already said, it is not a custom firmware but just a cloud replacement, meaning that even if 
- the robot had hardware specs that support linux
- you could root the robot
- you could run it on there and implement support for that Rowenta cloud communication,

it still would not change anything about the robot lacking a map and other things.

If you want to replace that unit with something that does run Valetudo, make sure to check the docs beforehand:
https://valetudo.cloud/pages/general/supported-robots.html

**Reply by ggirodda** [reply]
Ok, I was expecting this kind of answer 😥
Great work in any case @Hypfer, hopefully I will test Valetudo with my next Vacuum.
